l10n_be_partner_kbo_bce/tests/test_l10n_be_partner_kbo_bce.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2009-2020 Noviat.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestKboBceNumber(TransactionCase):
+    def setUp(self):
+        super(TestKboBceNumber, self).setUp()
+        self.rp_1 = self.env.ref("l10n_be_partner_kbo_bce.res_partner_1")
+        self.be = self.env.ref("base.be")
```

Comment:
Should use setUpClass, which is more performant.

Reply:
Sure but the test are not designed that way, it would demand to rewrite them.